packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v1-4-x/run-transaction-endpoint-v1.test.ts

```diff
@@ -122,11 +123,14 @@ test("runs tx on a Fabric v1.4.8 ledger", async (t: Test) => {
 
   const carId = "CAR277";
   const carOwner = uuidv4();
+  const fabricSigningCredential: FabricSigningCredential = {
+    keychainId,
+    keychainRef: keychainEntryKey,
+  };
 
   {
     const res = await apiClient.runTransactionV1({
-      keychainId,
-      keychainRef: keychainEntryKey,
+      fabricSigningCredential: fabricSigningCredential as FabricSigningCredential,
```

Comment:
@AzaharaC Does this not work (e.g. compile) if you just replace 

```typescript
fabricSigningCredential: fabricSigningCredential as FabricSigningCredential,
```

with

```typescript
fabricSigningCredential,
```

(so no casting and no property name duplication)

Reply:
done